### PR TITLE
Added support for Filipino language

### PR DESCRIPTION
**Pull request by mazodan**
Adding Filipino translation

### CODE REVIEW

**Comment by phelmig** [comment]
Hi @mazodan,
Thank you for your translation. Since we can't validate the contribution's quality we are currently requiring all translations to be reviewed by at least two native speakers (we count you as the contributor as the first). If everything is fine, a simple comment on this pull request is sufficient. If you can reach out to your personal network to find someone for a peer review it would be highly appreciated.

File: site/tl.njk [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                Nang walang pinagaralan ng paggamot at walang makakakuha ng gumaganang bakuna para sa kahit isang taon pa, ang mabisa na paraan para
+                kontrolin ang pandemyang coronavirus ay bigyan ang virus ng mas kaunting pagkakataon na kumalat. Ito ay mga listahan ng mga aksyon, sa uri ng
+                madaling ipatupad sa mas mabisa sa paglaban ng pandemya, ay dapat na sapat bilang isang nakatakdang ng maluwag na patnubay para sa mga tao na gustong sumali
+                sa kilusan at gumawa ng aksyon na maaring talagang gumawa ng pagkakaiba.
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Huwag matakot, pero maging alerto.</li>
+                <li class="pb-2">Maghugas ng kamay palagi at magsanay ng magandang kaugalian sa pag-ubo at pagbahin.</li>
+                <li class="pb-2">Subukang hawakan ang iyong mukha nang kaunti hangga't maaari, kasali ang iyong bunganga, ilong, at mata.</li>
```
Comment (engrleir) [inline site/tl.njk]: Mas ok siguro kung **Subukang limitahan and paghawak sa iyong mukha...** ung gagamiting statement. Nung unang basa ko kasi akala ko kailangang hawakan ko kahit paminsan minsan ung muka ko.

File: site/tl.njk [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                <li class="pb-2">Magsanay ng paghiwalay sa lipunan, walang yakap at halik, walang pakikikamay, walang apir. Kung kakailanganin, gamitin ang mas ligtas na alternatibo.</li>
+                <li class="pb-2">Huwag pumunta sa mga konsiyerto, dula, mga kaganapan sa palakasan, o kahit anong mga kaganapan sa libangan.</li>
+                <li class="pb-2">Pigilin ang sarili sa pagbisita sa museo, tanghalan, sinehan, night clubs, at sa mga ibang mga lugar ng paglibangan.</li>
+                <li class="pb-2">Lumayo sa mga pagtitipong pang pakikisalamuha and pagdiriwan, katulad ng mga pulong sa club, serbisyo sa relihiyon, at mga pribadong salu-salo.</li>
```
Comment (engrleir) [inline site/tl.njk]: "and pagdiriwan" should be "at pagdiriwang"

**Comment by engrleir** [comment]
btw. I'm a native speaker.

**Comment by mazodan** [comment]
Updated texts as suggested.

**Review by engrleir — APPROVED** [review]

**Review by phelmig — APPROVED** [review]

**Pull request merged**